docs/gaps/versioning-policy/README.md

```diff
@@ -0,0 +1,215 @@
+# gittuf Versioning Policy
+
+## Metadata
+
+* **Number:** TBD
+* **Title:** gittuf Versioning Policy
+* **Implemented:** No
+* **Withdrawn/Rejected:** No
+* **Sponsors:** Aditya Sirish A Yelgundhalli (adityasaky)
+* **Last Modified:** October 25, 2025
+
+## Abstract
+
+As gittuf is intended to be used in a distributed manner by developers
+contributing to a repository, we need to have a plan in place for version skews.
+It will inevitably be the case that different developers have different versions
+of the client. And, different versions of the client may support different
+versions of gittuf's policy and attestation schemas. If not handled with care,
+this can lead to unintended consequences, including degraded security
+guarantees.
+
+## Specification
+
+Currently, gittuf versions the following pieces:
+* attestation predicates: individual in-toto attestation predicates are
+  versioned in the predicate type URI; this is inherited from the in-toto
+  attestation specification
+* gittuf implementation: the implementation itself is versioned and encompasses
+  the behavior / workflows as well as the default versions of the metadata
+  generated by the client at that version
+
+The gittuf policy metadata is not versioned for its schema, which will be
+amended as part of this document. In summary, we intend to have the following
+gittuf components versioned:
+* attestation predicates
+* gittuf policy metadata
+* gittuf implementation and behavior
+
+Note that all versioning follows semantic versioning.
+
+### General Guidelines
+
+Whenever a client encounters a newer metadata version than that supported by the
+client, it must prompt the user to update the client. This is the case for minor
+and patch version bumps as well.
+
+### Client Workflows for Cross-Version Scenarios
+
+There are several aspects of version compatibility to consider.
+
+#### Attestation Predicate Versions
+
+##### Signing an older version of a predicate using a newer client
+
+If the predicate is of the same major version, the client must add a signature
+to the existing attestation. The client must validate the predicate during the
+signing flow using the validator for the version in question, i.e., the older
+version. The user with the older client should be prompted, out of band, to
+update their client.
+
+If the predicate is from a previous major version, the client must generate an
+equivalent predicate with the current major version. Out of band, the user who
+generated the older predicate version must be prompted to update their client
+and add their signature to the new attestation with the current predicate
+version.
+
+##### Signing a newer version of a predicate using an older client
+
+If the predicate is of the same major version, the client must add a signature
```

Comment:
Same initial sentence as prior para.  Should combine and mention this is always done.